types/eventsource/index.d.ts

```diff
@@ -3,8 +3,15 @@
 // Definitions by: Scott Lee Davis <https://github.com/scottleedavis>
 //                 Ali Afroozeh <https://github.com/afroozeh>
 //                 Pedro Gámez <https://github.com/snakedrak>
+//                 Akuukis <https://github.com/Akuukis>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.0
```

Comment:
Is there a particular reason the minimum required version needed to be updated? I'm not sure I see it.

Reply:
@uniqueiniquity Yes, `/// <reference lib="..." />` comes with [TS 3.0](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html). It is used in DOM test file to check whenever monkeypatch is mergable with the full DOM typings.

There is nothing that impacts types themselves.